python/sbp/client/loggers/base_logger.py

```diff
@@ -25,7 +25,7 @@ class BaseLogger(object):
   filename : string
     File to log to.
   """
-  def __init__(self, filename, mode="w", tags={}, dispatcher=dispatch):
+  def __init__(self, filename, mode="w", tags={}, dispatcher=None):
     self.handle = open(filename, mode)
```

Comment:
Yeah, that's a nice idea - a dispatcher thing that produces a rich message if dispatch succeeds.